src/components/ShareBlock/ShareBlock.tsx

```diff
@@ -68,90 +81,110 @@ const ShareBlock = ({
-                round={false}
-                bgStyle={{ fill: 'auto' }}
-              />
-            </FacebookShareButton>
-          </StyledShareButton>
-          <StyledShareButton
-            disableElevation
-            variant="contained"
-            color="#00acee"
-          >
-            <TwitterShareButton
-              url={url}
-              title={quote}
-              hashtags={[hashtag]}
-              beforeOnClick={() => {
-                trackShare('twitter');
-                return Promise.resolve();
-              }}
-            >
-              <TwitterIcon size={40} round={false} bgStyle={{ fill: 'auto' }} />
-            </TwitterShareButton>
-          </StyledShareButton>
-          <StyledShareButton
-            disableElevation
-            variant="contained"
-            color="#007fb1"
-          >
-            <LinkedinShareButton
-              url={url}
-              title={quote}
-              // @ts-ignore: seems to not be available for linkedin?
-              hashtags={[hashtag]}
-              beforeOnClick={() => {
-                trackShare('linkedin');
-                return Promise.resolve();
-              }}
-            >
-              <LinkedinIcon
-                size={40}
-                round={false}
-                bgStyle={{ fill: 'auto' }}
-              />
-            </LinkedinShareButton>
-          </StyledShareButton>
-          {isMatchingProjectionsRoute && (
-            <StyledShareButton
-              disableElevation
-              variant="contained"
-              color="#616161"
-              onClick={onClickEmbed}
-            >
-              Embed
-            </StyledShareButton>
-          )}
-        </ShareButtonContainer>
-      </ShareType>
-      <ShareType forceVertical={forceVertical}>
-        <ShareInstruction color="inherit" component="p" variant="subtitle2">
-          {newsletterInstruction ||
-            'Get the latest updates from the Covid Act Now team'}
-        </ShareInstruction>
-        <Newsletter county={countyName} stateId={stateId} />
-      </ShareType>
+    <ShareContainer>
+      <ShareRow newsletter={true}>
+        <ShareRowContentArea>
+          <NewsletterMockupWrapper>
+            <NewsletterMockup />
+          </NewsletterMockupWrapper>
+          <NewsletterTextArea>
+            <ShareInstructionHeader>
+              Receive daily updates
+            </ShareInstructionHeader>
+            <ShareInstructionBody>
+              We’ll email you fresh data and high-quality news related to
+              COVID-19.
+            </ShareInstructionBody>
+            <Newsletter county={countyName} stateId={stateId} />
+          </NewsletterTextArea>
+        </ShareRowContentArea>
+      </ShareRow>
+      <ShareRow newsletter={false}>
+        <ShareRowContentArea>
+          <SocialTextAreaWrapper>
+            <SocialTextArea>
+              <ShareInstructionHeader>
+                Share data, save lives
```

Comment:
this seems like a lot of div wrappers?

Reply:
Yeah, lots of flexing in alternating... but I don't think any are superfluous